### PR TITLE
S2t3

### PR DESCRIPTION
**Pull request by JamesWaterhouse**

### CODE REVIEW

File: src/scss/_desktop.scss [hunk 1]
```diff
@@ -0,0 +1,56 @@
+@media only screen 
+  and (min-width: 1024px)
+    {
+    body {
+        box-sizing: border-box;
```
Comment (richherbs) [inline src/scss/_desktop.scss]: should this be applied to * rather than body?

File: src/scss/_fonts.scss [hunk 2]
```diff
@@ -0,0 +1,2 @@
+$mainFont: 'Montserrat';
```
Comment (richherbs) [inline src/scss/_fonts.scss]: have a backup font?

File: src/scss/_mobile.scss [hunk 3]
```diff
@@ -0,0 +1,123 @@
+body {
+    background-color: $backgroundColor;
+    font-family: $mainFont;
```
Comment (richherbs) [inline src/scss/_mobile.scss]: backup font

File: src/scss/_mobile.scss [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+                    width: 100%;
+                        overflow: hidden;
+                    position: relative;
+                }
+            }
+            .dwellingInfo {
+                padding-top: 5%;
+                padding-bottom: 10%;
+                width: 90%;
+                font-family: $mainFont;
```
Comment (richherbs) [inline src/scss/_mobile.scss]: backup font

File: src/scss/styles.scss [hunk 5]
```diff
@@ -0,0 +1,7 @@
+@import 'colors';
+@import 'fonts';
+@import 'mobile';
+@import 'desktop';
+@import 'tablet';
+
+
```
Comment (richherbs) [inline src/scss/styles.scss]: remove whitespace

File: vendor/composer/LICENSE [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+
```
Comment (richherbs) [inline vendor/composer/LICENSE:21]: need to add vendor to git ignore

File: dist/css/styles.css [hunk 7]
```diff
@@ -0,0 +1,153 @@
+body {
+  background-color: #EDF1F2;
+  font-family: "Montserrat";
```
Comment (richherbs) [inline dist/css/styles.css:3]: backup font

**Review by richherbs — CHANGES_REQUESTED** [review]

File: dist/css/styles.css (hunk 7 above)
Comment (mgirar84) [reply dist/css/styles.css:3]: Thats css not sass

**Pull request merged**